lib/sqlalchemy/engine/base.py

```diff
@@ -1161,6 +1161,17 @@ def scalar(self, object_, *multiparams, **params):
 
         return self.execute(object_, *multiparams, **params).scalar()
 
+    def scalars(self, object_, *multiparams, **params):
+        """Executes and returns the first column of each row.
+
+        The underlying result/cursor is closed after execution.
+
```

Comment:
**mike bayer** (zzzeek) wrote:

let's also add the `:return:` for all new methods, in this case:

   :return: a :class:`_result.ScalarResult`

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3065